.github/workflows/test.yml

```diff
@@ -44,6 +44,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: ./
+      - name: Test Push
+        uses: ./
         with:
-          milliseconds: 1000
+          action: push
+          api-token: ${{ secrets.LOKALISE_API_TOKEN }}
+          project-id: 37600957607f701d17dfc3.52981513
```

Comment:
Would it make sense to actually make these a secret value too? Does it matter that much?

Reply:
Let's do that if you don't mind.  I just want to be extra cautious

Reply:
Done